CHANGELOG.md

```diff
@@ -926,6 +926,7 @@ Please be aware that Cortex `0.7.0` introduces some **breaking changes**. You're
 * [BUGFIX] Fixed leaked goroutines in the querier. #2070
 * [BUGFIX] Experimental TSDB: fixed `/all_user_stats` and `/api/prom/user_stats` endpoints when using the experimental TSDB blocks storage. #2042
 * [BUGFIX] Experimental TSDB: fixed ruler to correctly work with the experimental TSDB blocks storage. #2101
+ [BUGFIX] Fixed `Missing chunks and index config causing silent failure` Absence of chunks and index from schema config is not validated. #2731
```

Comment:
Please move this to the top of the file.

Reply:
@bboreham Sure! Thanks